.github/workflows/ci.yml

```
@@ -101,7 +112,10 @@ jobs:
           junit_family=legacy --durations=20
 
       - name: Test docs
-        if: matrix.python-version == '3.13'
+        if:
+          matrix.python-version == '3.13' && matrix.runs-on != "windows-latest"
```

actionlint (1.7.12)

[error] got unexpected character '"' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '. do you mean string literals? only single quotes are available for string delimiter

(expression)